app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -204,14 +230,26 @@ class Scene extends React.Component {
 
   render() {
     return (
-      <div
-        style={{
-          position: 'fixed',
-          top: 0,
-          left: 0,
-        }}
-        ref={this.scene}
-      />
+      <div>
+        <div
+          style={{
+            position: 'fixed',
+            top: 0,
+            left: 0,
+          }}
+          ref={this.scene}
+        />
+        <canvas
+          width={window.innerWidth}
+          height={window.innerHeight}
+          style={{
+            position: 'fixed',
+            top: 0,
+            left: 0,
+          }}
+          ref={this.overlay}
```

Comment:
Nit: this is technically an underlay.

Reply:
I think it is an overlay, as setting the background of the fallingBlocks to opaque canvas does not destroy this canvas, but the reverse is true. If the overlay's background is opaque, fallingBlocks is not visible. Unless I'm thinking about this backwards

Reply:
Oh derp. You're right. I got it backwards.